lib/dbms.js

```diff
@@ -80,16 +80,16 @@ const generate = async (modelPath) => {
   console.log(`Generate migration: ${driver}:${name} v${version} (${now})`);
   const previousPath = await getPreviousVersion(modelPath).catch(() => {});
   if (!previousPath) {
-    console.log('Previous version is not found in ../history');
-    return;
```

Comment:
I don't like void in this case

Reply:
- console.log returns undefined in node.js
- in other platforms it may differ
- this can't turn if-void-return into single line